platforms/karaf/features/src/main/resources/features.xml

```diff
@@ -114,6 +114,12 @@
     <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.java-apns/${java-apns-bundle-version}</bundle>
     <bundle>mvn:org.apache.camel/camel-apns/${project.version}</bundle>
   </feature>
+  <feature name='camel-asn1' version='${project.version}' resolver='(obr)' start-level='50'>
+    <feature version='${project.version}'>camel-core</feature>
```

Comment:
Looks like the format a little bit incorrect, please put <feature and <bundle on the same XML level

Reply:
thanks.